### PR TITLE
Add a new target KakuteH7mini

### PR DESCRIPTION
**Pull request by jamming**
Add a new target KakuteH7mini.
SPI FLASH BlackBox.
Standard 20mm  mounting Holes.

### CODE REVIEW

File: configs/default/HBRO-KAKUTEH7mini.config [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+# feature
+feature RX_SERIAL
+feature TELEMETRY
+feature OSD
+
+# serial
+serial 0 1 115200 57600 0 115200
+serial 5 64 115200 57600 0 115200
+serial 6 1024 115200 57600 0 115200
```
Comment (limonspb) [inline configs/default/HBRO-KAKUTEH7mini.config:121]: what are these Serials for?

File: configs/default/HBRO-KAKUTEH7mini.config [hunk 2]
```diff
@@ -0,0 +1,145 @@
+# Betaflight / STM32H743 (SH74) 4.2.11 Nov  9 2021 / 20:29:57 (948ba6339) MSP API: 1.43
+
+board_name KAKUTEH7MINI
```
Comment (limonspb) [inline configs/default/HBRO-KAKUTEH7mini.config:3]: board name does not match with the file name. Thats why build is crashing, check the build logs plz

**Review by limonspb — CHANGES_REQUESTED** [review]

File: configs/default/HBRO-KAKUTEH7mini.config (hunk 1 above)
Comment (jamming) [reply configs/default/HBRO-KAKUTEH7mini.config:121]: serial 0   --USB VCP
serial 5   --receiver
serial 6   --esc tel

**Comment by limonspb** [comment]
I guess now squash into 1 commit  and wait :)

**Comment by haslinghuis** [comment]
@jamming please squash

**Pull request closed without merging**